components/connector-service/internal/logging/contextlogger.go

```diff
@@ -0,0 +1,20 @@
+package logging
+
+import (
+	log "github.com/sirupsen/logrus"
+)
+
+func GetClusterLogger(tenant, group string) *log.Entry {
+	return log.WithFields(log.Fields{
+		"Group":  group,
+		"Tenant": tenant,
+	})
+}
+
+func GetApplicationLogger(application, tenant, group string) *log.Entry {
```

Comment:
We could simplify this method to something like that:
```go
func GetApplicationLogger(application, tenant, group string) *log.Entry {
	return GetClusterLogger(tenant, group).WithField("application", application)
}
```

Reply:
You are right! Done ;>